tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -563,3 +563,28 @@ if (!/\/2\.7/.test(Cypress.env('rancher_version')) && !/\/2\.8/.test(Cypress.env
     );
   });
 }
+
+describe('Test GitRepo Bundle do not show hash mismatch error.', { tags: '@p1'}, () => {
+
+  qase(195,
+
+    it("Fleet-195: Test GitRepo bundle hash is not mismatch.", { tags: '@fleet-195' }, () => {
+
+      const repoName = "test-bundle-hash-mistmatch"
+      const path = "qa-test-apps/bundle-hash-test"
+
+      cy.addFleetGitRepo({ repoName, repoUrl, branch, path });
+      cy.clickButton('Create');
+      cy.verifyTableRow(0, 'Active', repoName);
+      cy.checkGitRepoStatus(repoName, '1 / 1', '1 / 1');
+
+      // Verify that bundle is showing Active state and not showing hash mismatch error.
```

Comment:
This seems a regular gitrepo that expects to be ok.  A brief comment explaining why this is special or should be ok may help in the future if we need troubleshooting.

Reply:
Sure, I will add brief description about what is happening and link the issue for more information.